thorium/testsuite/test_thoriumflask.py

```diff
@@ -171,7 +195,20 @@ def test_get_with_pagination(self):
         body = json.loads(rv.data.decode())
         items = body['data']
         meta = body['meta']
-        self.assertDictEqual(meta, {'offset': 2, 'limit': 2})
+        print(meta)
+        self.assertDictEqual(
+            meta,
+            {
```

Comment:
This one is mostly good except for the comma thing. The rest should look like it